src/Common/Internal/Atom/Entry.php

```diff
@@ -242,7 +234,7 @@ public function setAuthor(array $author)
     /**
      * Gets the category.
      *
-     * @return array
+     * @return Category[]
      */
     public function getCategory()
```

Comment:
In this case, the property name should be "Categories". Anyway, you don't have to change it now